com.endlessm.Clippy.json

```diff
@@ -1,6 +1,5 @@
 {
   "id": "com.endlessm.Clippy",
-  "branch": "eos3",
   "runtime": "com.endlessm.apps.Platform",
```

Comment:
mergue and force push?

Reply:
Anyways I still not convinced we should ship this as an app, why cant we just add it to the runtime, so that apps can link directly with it, and use C API if we ever add such thing

Reply:
Because if you do that this will work only with apps that have been built with Endless' runtime. i.e. if you try to use Clippy with a Flathub app, you will have to override its runtime use (and that's done per app AFAIK, not globally).
Also, if you add this to the runtime, then the release of Clippy will be tied to the runtime's release, which, ATM would mean moving slowly.

So it's hacky but there are a few advantages to having it as an app.